docs/pages/cli-detach.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Detach symbols
+section: cli
+chapter: Command-line interface
+permalink: /cli/detach
+
+order: 104
+excerpt: Detach symbols from a Sketch document
+---
+
+Detach symbols from their symbol masters in a Sketch document with `sketchtool`. Recursively detaches symbols,
```

Comment:
Is it worth trying to explain the "detach" terminology from the ground up? Feels like there's a bit of assumed knowledge that it might be worthwhile to explicitly mention. Maybe it would help to firstly explain how normal Sketch documents manage symbols, before going on to describe what "detaching" does to that.